docs/_includes/navbar.html

```diff
@@ -49,15 +49,15 @@
 	  <ul class="nav navbar-nav">
 
 	    <li>
-	      <a href="index.html" class="{% if page.url contains 'index.html' %}active{% endif %}">Documentation</a>
+	      <a href="/index.html" class="{% if page.url contains 'index.html' %}active{% endif %}">Documentation</a>
```

Comment:
Please use relative paths. This will fail with our current docs setup because the base url is http://ci.apache.org/projects/flink/